statick_tool/reporting_plugin.py

```diff
@@ -28,3 +29,42 @@ def report(
     def set_plugin_context(self, plugin_context: Union[None, PluginContext]) -> None:
         """Setter for plugin_context."""
         self.plugin_context = plugin_context
+
+    def load_mapping(self) -> Dict[str, str]:
```

Comment:
since this is copied straight from the `ToolPlugin` class, does it make sense to extract a base StatickPlugin class now? Do you remember what the reason was for not having a base plugin class?